src/index.js

```diff
@@ -8,13 +10,13 @@ import {
   mapObject,
   mapObjectKeys,
   sanitizeAttributeName,
-  setBooleanAttribute,
-} from './util';
+  setBooleanAttribute
+} from "./util";
 
 const Types = {
-  bool: 'bool',
-  event: 'event',
-  json: 'json',
```

Comment:
Don't care about single vs double quotes, but removing the comma on last object entry is suboptimal for diffs

Reply:
I added prettier. And prettier has these defaults.

Reply:
Yeah I know, but that's the only setting I dislike and always set in my projects, because I think it's more beneficial to have readable diffs than saving a few lines of config ;)